ReSharper.FSharp/test/data/parsing/Expr - Const - Measure - Divide 01.fs.gold

```diff
@@ -0,0 +1,24 @@
+﻿Language: PsiLanguageType:F#
+IFSharpImplFile
+  IAnonModuleDeclaration
+    IExpressionStatement
+      IChameleonExpression
+        ILiteralExpr
+          FSharpToken(type:INT32, text:123)
+          IUnitOfMeasureClause
+            FSharpTokenType+LessTokenElement(type:LESS, text:<)
+            IDivideMeasure
+              INamedMeasure
+                INamedTypeUsage
+                  ITypeReferenceName
```

Comment:
I've just realized: perhaps a type usage wasn't needed here, and having just a type reference would be enough for features like Rename. We should probably try removing it when updating this code after FCS is updated.